doc/rst/language/spec/conversions.rst

```diff
@@ -276,6 +397,48 @@ Explicit conversions require a cast in the code. Casts are defined
 in :ref:`Casts`. Explicit conversions are supported between more
 types than implicit conversions, but not between all types.
 
```

Comment:
In the following, please add a close paren and remove the duplicate "two arguments".